Assets/Scripts/SceneManager.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+
+public class SceneManager : MonoBehaviour
+
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        // Check if there is a player in the scene
+        if (GameObject.Find("Player") == null)
```

Comment:
Probably would be better write/read state like this to GameManager, maybe like `GameManager.KillPlayer()`/`GameManager.IsPlayerKilled()` rather than checking individual game object state like this, but I think it's OK for now.